### PR TITLE
fix(core): unexpected toast style in editor

### PR DESCRIPTION
**Pull request by JimmFly**
image

### CODE REVIEW

**Comment by JimmFly** [comment]
* **#6924** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/6924?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @JimmFly and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/component/src/ui/scrollbar/index.css.ts [hunk 1]
```diff
@@ -30,6 +30,9 @@ export const scrollableViewport = style({
 globalStyle(`${scrollableViewport} > div`, {
   display: 'contents !important',
```
Comment (pengx17) [inline packages/frontend/component/src/ui/scrollbar/index.css.ts:31]: I forget why I added this. What will happen if this one is removed?

Comment (JimmFly) [reply packages/frontend/component/src/ui/scrollbar/index.css.ts:31]: Because there is a div with display:table under this viewport
image

Comment (pengx17) [reply packages/frontend/component/src/ui/scrollbar/index.css.ts:31]: Yeah I know. I mean I forget what will happen if using `display: table`.  At least it looks like when it is used, edgeless canvas will display (with height: 0)

**Review by pengx17 — APPROVED** [review]

**Pull request merged**